### ISSUE TITLE
What is the difference between Homebridge and HomeKit component plugins?

### ISSUE THREAD

**Issue by AlexKLMan** [home-assistant/homebridge-homeassistant#318]
I only just installed Hassbian on my Raspberry Pi so really new to all this.

I have some iOS devices and wish to control non-compatible Homekit devices with Siri, I understand Homebridge is the best option, which I have done before and failed, but now that I have Hassbian I think I should install a plugin of Homebridge instead of the full version.

However, now that I see this other option, https://www.home-assistant.io/components/homekit/
Which one should I use?

Thanks!

**Comment by mreassassin** [comment]
I'm in the same boat.  I was able to get everything working through homebridge except my xbox's.  That was the deal breaker for me.  I have it all set up correctly as far as I can tell.  now I'm trying home assistant out with no luck.  So far it see's all of the devices connected to my wink hub.  but nothing else.  basically just the xbox's and ps4.

I have all the numbers (IP addrss, Live ID, etc.) but don't know how to set this up.  is there an install per plugin like homebridge?

when I launch homeassistant (hass) I get a bunch of text flowing constantly.  In homebridge I only got text in the output when something was triggered.  Leads me to believe I'm doing something wrong.

any help appreciated!!
